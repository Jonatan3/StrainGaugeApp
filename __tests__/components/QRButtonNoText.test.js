import React from 'react';
import renderer from 'react-test-renderer';
import QRButtonNoText from '../../components/QRButtonNoText'

test('renders correctly', () => {
    const tree = renderer.create(<QRButtonNoText />).toJSON()
    expect(tree).toMatchSnapshot()
})