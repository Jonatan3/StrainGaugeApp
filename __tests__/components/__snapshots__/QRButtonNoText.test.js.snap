// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<View
  style={
    Object {
      "alignContent": "center",
      "flex": 1,
    }
  }
>
  <View
    accessible={true}
    collapsable={false}
    focusable={true}
    nativeID="animatedComponent"
    onClick={[Function]}
    onResponderGrant={[Function]}
    onResponderMove={[Function]}
    onResponderRelease={[Function]}
    onResponderTerminate={[Function]}
    onResponderTerminationRequest={[Function]}
    onStartShouldSetResponder={[Function]}
    style={
      Object {
        "opacity": 1,
      }
    }
  >
    <View
      style={
        Object {
          "alignItems": "center",
          "backgroundColor": "#2DA39D",
          "borderBottomWidth": 2,
          "borderColor": "#00817C",
          "borderRadius": 100,
          "borderRightWidth": 1,
          "flexDirection": "row",
          "justifyContent": "center",
        }
      }
    >
      <View
        style={
          Object {
            "margin": 10,
          }
        }
      >
        <RNSVGSvgView
          align="xMidYMid"
          bbHeight="36px"
          bbWidth="36px"
          enable-background="new 0 0 24 24"
          fill="#FFFFFF"
          focusable={false}
          height="36px"
          meetOrSlice={0}
          minX={0}
          minY={0}
          style={
            Array [
              Object {
                "backgroundColor": "transparent",
                "borderWidth": 0,
              },
              Object {
                "flex": 0,
                "height": 36,
                "width": 36,
              },
            ]
          }
          vbHeight={24}
          vbWidth={24}
          width="36px"
          xmlns="http://www.w3.org/2000/svg"
        >
          <RNSVGGroup
            fill={4294967295}
            propList={
              Array [
                "fill",
              ]
            }
          >
            <RNSVGRect
              fill={null}
              height="24"
              propList={
                Array [
                  "fill",
                ]
              }
              width="24"
              x={0}
              y={0}
            />
            <RNSVGPath
              d="M9.5,6.5v3h-3v-3H9.5 M11,5H5v6h6V5L11,5z M9.5,14.5v3h-3v-3H9.5 M11,13H5v6h6V13L11,13z M17.5,6.5v3h-3v-3H17.5 M19,5h-6v6 h6V5L19,5z M13,13h1.5v1.5H13V13z M14.5,14.5H16V16h-1.5V14.5z M16,13h1.5v1.5H16V13z M13,16h1.5v1.5H13V16z M14.5,17.5H16V19h-1.5 V17.5z M16,16h1.5v1.5H16V16z M17.5,14.5H19V16h-1.5V14.5z M17.5,17.5H19V19h-1.5V17.5z M22,7h-2V4h-3V2h5V7z M22,22v-5h-2v3h-3v2 H22z M2,22h5v-2H4v-3H2V22z M2,2v5h2V4h3V2H2z"
            />
          </RNSVGGroup>
        </RNSVGSvgView>
      </View>
    </View>
  </View>
</View>
`;
